build.go

```diff
@@ -52,7 +53,13 @@ func main() {
 }
 
 func VetActionFunc(_ *cli.Context) error {
-	return exec.Command("go", "vet").Run()
+	cmd := exec.Command("go", "vet")
+
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
```

Comment:
We may write a wrapping function for this recurring pattern, but this is a non-blocking nit pick.

Reply:
Also, I assume there should be some function exec that sets up the process this way. I find it strange that this is not the default.

Reply:
I also find it strange that there isn't a function in that package that does this by default. Here is the documentation => https://golang.org/pkg/os/exec/#Command

Reply:
eek, horrible.

Reply:
https://github.com/urfave/cli/pull/873/commits/29ad6ee6ad7e02dbec3334e8843bb6711c011b55 ✔️